primary/src/block_remover.rs

```diff
@@ -557,9 +557,9 @@ impl BlockRemover {
             let message = PrimaryWorkerMessage::DeleteBatches(batch_ids.clone());
 
             // send the request
-            self.worker_network
-                .unreliable_send(worker_name.clone(), &message)
-                .await;
+            let _ = self
```

Comment:
what does the `let _ =` do? Asking mostly for my own edification :)

Reply:
It immediately drops the response and in this case silences the warning well now get for ignoring a Result type (since most of these call sites don't presently care about the result anyway)

Reply:
Cool makes sense! Didn't realize that without that it'd emit a warning

Reply:
you can annotate types with `#[must_use]` to have the compiler throw a warning if a type isn't used. its super helpful generally.